app.rb

```diff
@@ -0,0 +1,103 @@
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'rental'
+
+class App
+  attr_accessor :books, :people, :rentals
+  attr_reader :main
+
+  def initialize(books = [], people = [], rentals = [])
+    @books = books
+    @people = people
+    @rentals = rentals
+  end
+
+  def book_list
+    puts 'Books list'
+    @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+  end
+
+  def people_list
+    puts 'People list'
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp.to_i
+    case person_type
+    when '1' then create_student
+    when '2' then create_teacher
+    else
+      puts 'Invalid selection'
+      create_person
+    end
+    puts 'Person created successfully'
+  end
+
+  def create_student
+    puts 'Age: '
+    age = gets.chomp.to_i
+    puts 'Name: '
+    name = gets.chomp
+    puts 'Do you have permission[Y/N]: '
+    parent_permission = gets.chomp
+    student = Student.new(age, name, parent_permission: parent_permission)
```

Comment:
- [ ] There is an error that occurs when I try to insert the Y or N, therefore it stops the student from being created. Please look this up 🙏

Reply:
Done, it wsa hard to find that easy solution